### PR TITLE
TOOLS/PERF: add test over stream API

### PR DESCRIPTION
**Pull request by evgeny-leksikov**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4852/ for details (Mellanox internal link).

**Comment by evgeny-leksikov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4853/ for details (Mellanox internal link).

**Comment by evgeny-leksikov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4854/ for details (Mellanox internal link).

File: src/tools/perf/perftest.c [hunk 1]
```diff
@@ -139,7 +139,10 @@ test_type_t tests[] = {
     {"ucp_cswap", UCX_PERF_API_UCP, UCX_PERF_CMD_CSWAP, UCX_PERF_TEST_TYPE_STREAM_UNI,
      "UCP atomic compare-and-swap latency / bandwidth / message rate"},
 
-    {NULL}
+    {"stream_bw", UCX_PERF_API_UCP, UCX_PERF_CMD_STREAM, UCX_PERF_TEST_TYPE_STREAM_UNI,
```
[the remark this answers is not in the document]
Comment (evgeny-leksikov) [reply src/tools/perf/perftest.c]: done

File: src/tools/perf/perftest.c [hunk 2]
```diff
@@ -139,7 +139,10 @@ test_type_t tests[] = {
...
+     "UCP stream bandwidth"},
```
[the remark this answers is not in the document]
Comment (evgeny-leksikov) [reply src/tools/perf/perftest.c]: done

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4872/ for details (Mellanox internal link).

**Pull request merged**